examples/experimental/i3s-17-and-debug/coloring-utils.js

```diff
@@ -20,33 +21,54 @@ export const DEPTH_COLOR_MAP = {
   12: [125, 58, 174]
 };
 
-export const DEPTH_MAX_LEVEL = 12;
-
-export function getTileColor(tile, options) {
-  switch (options.coloredBy) {
-    case COLORED_BY.TILE:
-      return getColorByTile(tile.id, options.colorsMap);
-    case COLORED_BY.DEPTH:
-      return getColorByDepth(tile.depth);
-    case COLORED_BY.CUSTOM:
-      return getCustomColor();
-    default:
-      return [255, 255, 255];
+const DEPTH_MAX_LEVEL = 12;
+const DEFAULT_COLOR = [255, 255, 255];
+
+export default class ColorMap {
```

Comment:
If the main export is the ColorMap class, the file should be called `color-map.js`

Reply:
Done